acceptance/powershell/terraform/application.tf

```diff
@@ -0,0 +1,58 @@
+data "aws_ami" "windows_ami" {
+  most_recent = true
+  filter {
+    name = "owner-alias"
+    values = ["amazon"]
+  }
+  filter {
+    name = "name"
+    values = ["Windows_Server-2012-R2*-English-*-Base-*"]
+  }
+  filter {
+    name = "architecture"
+    values = ["x86_64"]
+  }
+  filter {
+    name = "virtualization-type"
+    values = ["hvm"]
+  }
+  filter {
+    name = "block-device-mapping.volume-type"
+    values = ["gp2"]
+  }
+  filter {
+    name = "image-type"
+    values = ["machine"]
+  }
+}
+
+resource "aws_instance" "mixlib_install_ps1" {
+  count = 1
+
+  ami           = "${data.aws_ami.windows_ami.id}"
+  instance_type = "${var.aws_instance_type}"
+  key_name      = "es-infrastructure"
+
+  subnet_id         = "subnet-19ac017c" # Planet Releng Private Subnet
+  source_dest_check = false
+
+  vpc_security_group_ids = [
+    "sg-96274af3",
+  ]
+
+  # TODO: What needs to happen here?
```

Comment:
@schisamo This is here I'm blocked right now. I've tried all sorts of configurations. Will need to pair on this
```
    STDERR: Error applying plan:

    1 error(s) occurred:

    * unknown error Post http://10.194.11.185:5985/wsman: dial tcp 10.194.11.185:5985: getsockopt: operation timed out

    Terraform does not automatically rollback in the face of errors.
    Instead, your Terraform state file has been partially updated with
    any resources that successfully completed. Please address the error
    above and apply again to incrementally change your infrastructure.
```